components/AppBar.tsx

```diff
@@ -18,7 +18,7 @@ const AppBar = (props: PropsWithChildren<{
 
 	return (
 		<>
-			<header ref={headerRef} className={props.position + " w-full box-border z-20 top-0 right-0 left-auto shrink-0 print:absolute border-b-secondary-dark border-b-2 " + props.backdropClassName}>
+			<header ref={headerRef} className={`${props.position} w-full box-border z-20 top-0 right-0 left-auto shrink-0 print:absolute border-b-border-dark border-b-2  ${props.backdropClassName}`}>
```

Comment:
Just for the sake of my curiosity, what does this change do?

Reply:
It was something that CodeRabbit was recommending for improving readability. I did it for a few cases, but it was just polluting the pull request; so I didn't carry on.

Reply:
This change also changes the bottom border color based on the updated definitions of the colors in our `tailwind.config.js` file.